config.js

```diff
@@ -1,5 +1,6 @@
-{
+module.exports = {
```

Comment:
Converted this file to JS as Node was choking on regexs in JSON. Also allows for comments.